media-plugins/gst-plugins-jack/gst-plugins-jack-1.18.4-r1.ebuild

```diff
@@ -10,5 +10,5 @@ DESCRIPION="JACK audio server source/sink plugin for GStreamer"
 KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
 IUSE=""
 
-RDEPEND="virtual/jack[${MULTILIB_USEDEP}]"
+RDEPEND="media-sound/jack2[${MULTILIB_USEDEP}]"
```

Comment:
This would exclude media-video/pipewire[jack-sdk] which has the same headers as JACK2 and while not all functions are implemented, is likely to work, too. Therefore please consider using this instead:

        || (
                media-sound/jack2[${MULTILIB_USEDEP}]
                media-video/pipewire[${MULTILIB_USEDEP},jack-sdk(-)]
        )